docs/source/citation.rst

```diff
@@ -0,0 +1,49 @@
+.. _acknowledgement:
+
+Acknowledgement
+===============
+
+openPMD (Open Standard for Particle-Mesh Data Files) is a community project with many people contributing to it.
+If you use openPMD and/or openPMD related software in your work, please credit it when publishing and/or presenting work performed with it in order to give back to the community.
+
+openPMD-standard
+----------------
+
+The central definition of openPMD is the meta-data schema defined in ... .
+The most general reference to openPMD is:
+
+    Axel Huebl, Remi Lehe, Jean-Luc Vay, David P. Grote, Ivo F. Sbalzarini, Stephan Kuschel, David Sagan, Christopher Mayes, Frédéric Pérez, Fabian Koller, and Michael Bussmann.
+    "openPMD: A meta data standard for particle and mesh based data,"
+    `DOI:10.5281/zenodo.591699 <https://doi.org/10.5281/zenodo.591699>`_ (2015)
+
+Since the openPMD-standard is an actively evolving meta-data schema, a specific version of the openPMD standard might be used in your work.
+You can select a version-specific DOI from the `release page <https://github.com/openPMD/openPMD-standard/releases>`_ and add the version number to the cited title, e.g.
+
+    ... (author list as above) ...
+    "openPMD 1.1.0: A meta data standard for particle and mesh based data," `DOI:10.5281/zenodo.1167843 <https://doi.org/10.5281/zenodo.1167843>`_ (2018)
+
+openPMD-api
+-----------
+
+openPMD-api is a reference implementation of the openPMD-standard for popular data formats.
+
+openPMD-api is a scientific software and cited generally as
+
+    Fabian Koller, Franz Poeschel, Junmin Gu, and Axel Huebl.
+    "openPMD-api: C++ & Python API for Scientific I/O with openPMD"
+    `DOI:10.14278/rodare.27 <https://doi.org/10.14278/rodare.27>`_ (2018)
+
+As a software citation, you almost always want to refer to a specific version of openPMD-api in your works:
```

Comment:
This is a great PR @ax3l ! And it really helps users.
I was wondering if it would be interesting to explain why it is so useful to know the code versions (not just of openPMD-api, but of all those used). Feel free to not like my suggestion, but I'm thinking of something in the lines of:
```suggestion
It is good scientific practice to document all software versions in, e.g. a methods section, so you, reviewers and collaborators can reproduce your results.
In a software citation, you almost always want to refer to a specific version of openPMD-api in your work, as illustrated below.
```
What do you think?

Reply:
That's a great point, thank you. I like giving reason with guidelines, yay!